app/src/main/java/org/fossasia/openevent/general/search/SearchLocationViewModel.kt

```diff
@@ -23,11 +33,44 @@ class SearchLocationViewModel(
     val eventLocations: LiveData<List<EventLocation>> = mutableEventLocations
     private val mutableShowShimmer = MutableLiveData<Boolean>()
     val showShimmer: LiveData<Boolean> = mutableShowShimmer
+    private var geoCodingRequest: MapboxGeocoding? = null
```

Comment:
This should be a list of geocoding requests

Reply:
What if I cancel the previous request before enqueue new request ?

Reply:
That works as well